jetty-ee10/jetty-ee10-maven-plugin/src/main/java/org/eclipse/jetty/ee10/maven/plugin/AbstractUnassembledWebAppMojo.java

```diff
@@ -176,7 +176,7 @@ protected void configureUnassembledWebApp() throws IOException
             if (webXml != null)
             {
                 Resource r = webApp.getResourceFactory().newResource(webXml.toPath());
-                if (r.exists() && !r.isDirectory())
+                if (r != null && !r.isDirectory())
```

Comment:
Maybe readability would be better if we had some static methods that protected for null values:

```suggestion
                if (Resource.isDirectory(r))
```

We could avoid scattering lots of null checks all over the code then.